kubernetes/GKE/README.md

```diff
@@ -66,7 +81,15 @@ git clone https://github.com/pytorch/serve.git
 cd serve/kubernetes/GKE
 ```
 
-#### 2.2 Install NVIDIA device plugin
+**_NOTE:_** By default the helm chart installs GPU version of torchserve. Follow steps in section [2.2](####-2.2-For-CPU-setup) for running in a CPU only cluster. For GPU setup section [2.2](####-2.2-For-CPU-setup) can be skipped.
+
+#### 2.2 For CPU setup
+
+* Change torchserve image in Helm/values.yaml to the CPU version
+* Set `n_gpu` to `0` in Helm/values.yaml
+* Skip NVIDIA plugin installation in section [2.3](#####-2.3-Install-NVIDIA-device-plugin)
+  
+#### 2.3 Install NVIDIA device plugin
 
 Before the GPUs in the nodes can be used, you must deploy a DaemonSet for the NVIDIA device plugin. This DaemonSet runs a pod on each node to provide the required drivers for the GPUs.
```

Comment:
Add a note that this is only needed if deploying for GPU nodes

Reply:
By default torchserve used GPU, i have added in section 2.2 CPU setup we have to set `n_gpu` to `0` and skip nvidia daemoset.